sample_files/sample_CSV/test_grades.py

```diff
@@ -0,0 +1,61 @@
+# import requests
+# from requests.auth import HTTPBasicAuth
+from verify import parse_assigned_zones, get_actual_zone_dict
+
+actual_dict = get_actual_zone_dict()
+assigned_dict = parse_assigned_zones()
+
+output_list = []
+file_list = ['TestExam_1']
+for file in file_list:
+    with open('%s.csv' % (file, ), 'r') as zones:
+        # Get header row contents
+        header_str = zones.readline()
+
+        line_list = zones.readlines()
+
+        # line_list = line_list[:1]
+        for index, line in enumerate(line_list):
+            line = line.strip()
+            if len(line) > 0 and 'Missing' not in line:
+                temp_array = []
+                record = line.split(',')
+                student_rcs = record[1]
+                student_name = record[0]
+                student_tmp = student_name.rsplit(' ', 1)
```

Comment:
Yeah, I talked with Barb on this. We agreed that breaking up the name isn't too important since we're not using the field in the upload. So I only broke it up as the last word in a name to be the last name.